cob_hardware_config/raw3-5/config/base_controller.yaml

```diff
@@ -20,17 +21,21 @@ twist_controller:
       spring: 15.0 #10.0
       damp: 2.75 #2.5
       virt_mass: 0.1 #0.1
-      d_phi_max: 12.0 #100.0
+      d_phi_max: 10.0 #100.0
       dd_phi_max: 40.0 #100.0
   wheels: &wheels
     - steer: fl_caster_rotation_joint
       drive: fl_caster_r_wheel_joint
+      steer_neutral_position: -180.0
```

Comment:
> Only one wheel was driving in the correct direction

The neutral postion position should not affect driving.
It is just a default target for the steer wheels after start-up.